### PR TITLE
New bluetooth module

### PR DESCRIPTION
**Pull request by jmdana**
I wrote a new module which shows the status of the bluetooth connection ("BT: OFF" vs. "BT: \<DEVICE_NAME\>").

### CODE REVIEW

**Comment by ultrabug** [comment]
Thanks for this nice contribution @jmdana !

I'll permit myself some QA annotations on the code you're submitting. I try to keep the modules as much alike as they can be. If you don't have time or cba doing it, I'll take the liberty to do it after the merge so there's nothing mandatory here, just say so ;)

Cheers mate

**Comment by jmdana** [comment]
Sure. Please make the annotations in the code and I will have a look at them.

File: py3status/modules/bluetooth.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    - hcitool
+    - sed
+
+@author jmdana <https://github.com/jmdana>
+@license GPLv3 <http://www.gnu.org/licenses/gpl-3.0.txt>
+"""
+
+from time import time
+import subprocess
+import shlex
```
Comment (ultrabug) [inline py3status/modules/bluetooth.py]: please, order your imports alphabetically

File: py3status/modules/bluetooth.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    color_good = None
+    color_bad = None
+
+    def __init__(self):
+        pass
+
+    def kill(self, i3s_output_list, i3s_config):
+        pass
+
+    def on_click(self, i3s_output_list, i3s_config, event):
```
Comment (ultrabug) [inline py3status/modules/bluetooth.py]: __init__, kill and on_click should be removed if they don't actually do anything please

File: py3status/modules/bluetooth.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        pass
+
+    def get_status(self, i3s_output_list, i3s_config):
```
Comment (ultrabug) [inline py3status/modules/bluetooth.py]: convention is to name the main module's method as the module name (simply bluetooth in your case)

File: py3status/modules/bluetooth.py [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    """
+    from time import sleep
+    x = Py3status()
+    config = {
+        "color_good": "#00FF00",
+        "color_bad": "#FF0000",
+        }
+
+    while True:
+        print(x.get_status([], config))
```
Comment (ultrabug) [inline py3status/modules/bluetooth.py]: obvious renaming as per above comment, cheers

**Comment by jmdana** [comment]
Done.

I've also removed the dependency on "sed" so it uses Python regular expressions, just like I did for keyboard_layout.

Please keep the good work, this is a great project!

**Comment by ultrabug** [comment]
Thanks a lot @jmdana, I'm always glad to get some nice contributions !

On a future enhancement of this module, maybe a format configuration parameter would be nice :)

**Comment by jmdana** [comment]
Just point me to a module that I can use as a template (so we make sure that I get the right one) and I'll have a look at it.

**Comment by ultrabug** [comment]
The net_rate module is one of the most configurable one. I find the placeholders and format very elegant, you surely will too :)

https://github.com/ultrabug/py3status/blob/master/py3status/modules/net_rate.py

**Pull request merged**